code/datums/craft/recipes/weapon.dm

```diff
@@ -236,3 +236,16 @@
 		list(CRAFT_MATERIAL, 2, MATERIAL_GLASS, "time" = 10),
 		list(QUALITY_ADHESIVE, 15, 70)
 	)
+
+
+/datum/craft_recipe/weapon/stungloves //Occulus edit
```

Comment:
Why's this not modular!

Reply:
Because it can’t be modular. The code for the crafting recipe menu pulls datums from specific files to list them in the menu.

Reply:
> Because it can’t be modular. The code for the crafting recipe menu pulls datums from specific files to list them in the menu.

We have several unique crafting recipies that are modular. Disktoasters and sofas specifically. It should be perfectly possible to make this modular.